controllers/packetmachine_controller.go

```diff
@@ -173,15 +173,17 @@ func (r *PacketMachineReconciler) reconcile(ctx context.Context, machineScope *s
 	// If the PacketMachine doesn't have our finalizer, add it.
 	controllerutil.AddFinalizer(packetmachine, infrastructurev1alpha3.MachineFinalizer)
 
-	if !machineScope.Cluster.Status.InfrastructureReady {
-		machineScope.Info("Cluster infrastructure is not ready yet")
-		return ctrl.Result{Requeue: true, RequeueAfter: 30 * time.Second}, nil
-	}
+	if !machineScope.IsControlPlane() {
+		if !machineScope.Cluster.Status.InfrastructureReady {
```

Comment:
This should be reverted. Nothing starts until cluster is ready, APIEndpoint added when ready.